### PR TITLE
feat:added product ordered price , fix: remove from cart feature 

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Cart Page
I have added ordered price of product in  the cart page . So ignore other files. 

Files to check
src/components/cart-card/CartCard.jsx
src/pages/cart-page/Cart.js
src/context/WishListContext.js
PS - Ignore other files. .

What not to check
Responsiveness

image

### CODE REVIEW

File: src/context/WishListContext.js [hunk 1]
```diff
@@ -5,47 +5,89 @@ const WishListContext = createContext(null);
 const WishListProvider = ({children}) => {
     
     //-------------------for wishlist------------------------- 
-    const [ wishlist, setWishList ] = useState(JSON.parse(localStorage.getItem ("wishlist")));
+    const [ wishlist, setWishList ] = useState(localStorage.getItem ("wishlist")? JSON.parse(localStorage.getItem("wishlist")):[]);
```
Comment (arjun-computer-geek) [inline src/context/WishListContext.js:8]: It's good that you have predict the error here. Good job

File: src/context/WishListContext.js [hunk 2]
```diff
@@ -5,47 +5,89 @@ const WishListContext = createContext(null);
...
     //using function removing product from wishlist
     const removeFromWishList = (id) => {
 
         setWishList(wishlistProducts => wishlistProducts.filter(item => item._id !== id)  )
     }
 
-
     //--------------------for AddToCart------------------------
-  
     const [cartItem , setcartItem] = useState(JSON.parse(localStorage.getItem ("cartItem")));
```
Comment (arjun-computer-geek) [inline src/context/WishListContext.js:34]: It will be more scoring if you will use reducer instead state

File: src/context/WishListContext.js [hunk 3]
```diff
@@ -5,47 +5,89 @@ const WishListContext = createContext(null);
...
+        cartItem.map(cartItem => totalPrice += cartItem.discountedPrice * cartItem.quantity)
+        setTotal(totalPrice);
+        console.log(totalPrice)
+
+    }
+    useEffect(()=> {
+        calculateTotal();
+    },[cartItem])
+
+    //storing cart product in local storage
```
Comment (arjun-computer-geek) [inline src/context/WishListContext.js:51]: Leaving meaning full comment is good practice. Great

**Review by arjun-computer-geek — APPROVED** [review]
overall fully managed code. good folder structure. Readable code. Good job @Shubhamkr2610

**Comment by santhoshpemmaka** [comment]
Small Pr. No suggestion from my side. All looking good. Great job. Good to merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/context/WishListContext.js (hunk 2 above)
Comment (Shubhamkr2610) [reply src/context/WishListContext.js:34]: okay , I will try

**Pull request merged**